lib/tableau/config.ex

```diff
@@ -5,6 +5,7 @@ defmodule Tableau.Config do
 
   defstruct [
     :url,
+    base_path: "",
```

Comment:
Please add documentation for this. Its in the Tableau moduledoc

Reply:
done